pkgs/development/python-modules/frida-python/default.nix

```diff
@@ -0,0 +1,47 @@
+{
+  lib,
+  stdenv,
+  fetchurl,
+  pkgs,
+  typing-extensions,
+}: let
+  pname = "frida-python";
+  version = "16.0.19";
+  namePypi = "frida";
+
+  devkit = fetchurl {
```

Comment:
this should go to `passthru`

Reply:
I couldn't figure out how to access it from the `postPatch` when it was in `passthru`, so I still define it in the `let`, and then use it in `postPatch` and pass it thru passthru too.